### PR TITLE
Move autocomplete JS and CSS to top-level layout.html

### PR DESCRIPTION
**Pull request by gheinrich**

### CODE REVIEW

File: digits/templates/layout.html [hunk 1]
```diff
@@ -46,6 +47,21 @@
...
 
+<style type="text/css">
+.autocomplete-suggestions { border: 1px solid #999; background: #FFF; overflow: auto; }
+.autocomplete-suggestion { padding: 2px 5px; white-space: nowrap; overflow: hidden; }
+.autocomplete-selected { background: #F0F0F0; }
+.autocomplete-suggestions strong { font-weight: normal; color: #3399FF; }
+.autocomplete-group { padding: 2px 5px; }
+.autocomplete-group strong { display: block; border-bottom: 1px solid #000; }
+</style>
+
```
Comment (lukeyeager) [inline digits/templates/layout.html]: Can we move these into `digits/static/css/style.css`?

**Comment by lukeyeager** [comment]
Should we make the class something like `.autocomplete_path` so we could add autocomplete for other things later (like `.autocomplete_job_id` or something)?

**Comment by lukeyeager** [comment]
Can you also add this class to `custom_network_snapshot` on "classificationModel#new" and "genericModel#new" and to all the fields at "genericDataset#new"?

I was going to create a pull request to do this, but I would be stepping on your work here too much. I'll just let you do it.

**Comment by gheinrich** [comment]
Hi Luke, thanks for the comments, I will propose another patch.

**Comment by lukeyeager** [comment]
Looks great, thanks!

**Pull request merged**